worlds/khrecom/Client.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import annotations
+import os
+import sys
+import asyncio
+import shutil
+import logging
+import time
+from calendar import timegm
+
+import ModuleUpdate
+ModuleUpdate.update()
+
+import Utils
+
+death_link = False
+
+item_num = 1
+
+logger = logging.getLogger("Client")
+
+if __name__ == "__main__":
+    Utils.init_logging("KHRECOMClient", exception_logger="Client")
+
+from NetUtils import NetworkItem, ClientStatus
+from CommonClient import gui_enabled, logger, get_base_parser, ClientCommandProcessor, \
+    CommonContext, server_loop
+
+
+def check_stdin() -> None:
+    if Utils.is_windows and sys.stdin:
+        print("WARNING: Console input is not routed reliably on Windows, use the GUI instead.")
+
+class KHRECOMClientCommandProcessor(ClientCommandProcessor):
+    def __init__(self, ctx):
+        super().__init__(ctx)
+
+    def _cmd_deathlink(self):
+        """Toggles Deathlink"""
+        global death_link
```

Comment:
You should avoid globals wherever possible. I would make deathlink an instance variable of your client, unless you have a reason that it must be accessible elsewhere.